docs/crocks/Star.md

```diff
@@ -0,0 +1,61 @@
+# Star
+
+`Monad m => Star a (m b)`
```

Comment:
I thought the new line could better say "under the implication of", the `.` was just a poopie way of indenting.  Great jorb never less!!